awx/main/utils/plugins.py

```diff
@@ -56,4 +74,12 @@ def load_combined_inventory_source_options() -> dict[str, str]:
 
     plugins = compute_cloud_inventory_sources()
 
-    return dict(zip(plugins, plugins), file='file')
+    plugin_description = discover_available_cloud_provider_descriptions()
+
+    plugin_description['scm'] = 'Sourced from a Project'
```

Comment:
Since this is a plugin we don't want to assume that the `ssh` plugin will be discovered. Change this to something like:

```
if 'scm' in plugin_description:
    plugin_description['scm'] = 'Sourced from a Project'
```

Reply:
@chrismeyersfsu since we are manually adding scm and constructed in `compute_cloud_inventory_sources`  those two are always present- Does that need to change? Maybe I'm thinking about that wrong

Reply:
Thanks for pointing this out. You are correct. Marking this comment resolved